models/intel/handwritten-simplified-chinese-recognition-0001/description/handwritten-simplified-chinese-recognition-0001.md

```diff
@@ -0,0 +1,47 @@
+# handwritten-simplified-chinese-recognition-0001
+
+## Use Case and High-Level Description
+
+This is a network for handwritten simplified Chinese text recognition scenario. It consists of a VGG16-like backbone,
+reshape layer and a fully connected layer.
+The network is able to recognize simplified Chinese text consisting of characters in the [SCUT-EPT](https://github.com/HCIILAB/SCUT-EPT_Dataset_Release).
+
+## Example
+
+![](./test.png) -> 的人不一了是他有为在责新中任自之我们
+
+## Specification
+
+| Metric                                         | Value              |
+|------------------------------------------------|--------------------|
+| GFlops                                         |                    |
+| MParams                                        |                    |
```

Comment:
Are you going to fill these in?

Reply:
For the previous Japanese model, Vladimir helped fill in these values. I'm not sure how I should compute these values.

Reply:
You can use the Model Analyzer, which you can find in the `deployment_tools/tools/workbench/model_analyzer` directory within OpenVINO toolkit.

Reply:
Thanks for the reply. Should I use FP32 IR or FP16 IR when using the model_analyzer?

Reply:
I don't think it will matter. But FP32, I guess.